src/common/codeUtil.ts

```diff
@@ -48,4 +51,85 @@ export class CodeUtil {
 
         return regs;
     }
+
+    /** Align variable assignment by = operator in selected text
+     * @param selection Text selection in editor
+     */
```

Comment:
Please document the return value, I think it's just the aligned selection, right?

```suggestion
     * @returns aligned selection
     */
```